src/main/resources/db/migration/v0.31.0.4__agm_phenotype_annotation.sql

```diff
@@ -0,0 +1,128 @@
+CREATE INDEX phenotypeannotation_relation_index ON phenotypeannotation USING btree(relation_id);
+
+CREATE TABLE phenotypeannotation_ontologyterm (
+	phenotypeannotation_id bigint,
+	phenotypeterms_id bigint
+);
+
+ALTER TABLE phenotypeannotation_ontologyterm ADD CONSTRAINT phenotypeannotation_ontologyterm_phenotypeannotation_id_fk FOREIGN KEY (phenotypeannotation_id) REFERENCES phenotypeannotation (id);
+ALTER TABLE phenotypeannotation_ontologyterm ADD CONSTRAINT phenotypeannotation_ontologyterm_phenotypeterms_id_fh_fk FOREIGN KEY (phenotypeterms_id) REFERENCES ontologyterm (id);
+CREATE INDEX phenotypeannotation_ontologyterm_phenotypeannotation_index ON phenotypeannotation_ontologyterm USING btree (phenotypeannotation_id);
+CREATE INDEX phenotypeannotation_ontologyterm_phenotypeterms_index ON phenotypeannotation_ontologyterm USING btree (phenotypeterms_id);
+
+CREATE TABLE agmphenotypeannotation (
+	id bigint PRIMARY KEY,
+	phenotypeannotationsubject_id bigint,
+	assertedallele_id bigint,
+	inferredallele_id bigint,
+	inferredgene_id bigint
+);
+
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_id_fk FOREIGN KEY (id) REFERENCES phenotypeannotation (id);
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_phenotypeannotationsubject_id_fk FOREIGN KEY (phenotypeannotationsubject_id) REFERENCES affectedgenomicmodel (id);
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_assertedallele_id_fk FOREIGN KEY (assertedallele_id) REFERENCES allele (id);
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_inferredallele_id_fk FOREIGN KEY (inferredallele_id) REFERENCES allele (id);
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_inferredgene_id_fk FOREIGN KEY (inferredgene_id) REFERENCES gene (id);
+CREATE INDEX agmphenotypeannotation_phenotypeannotationsubject_index ON agmphenotypeannotation USING btree (phenotypeannotationsubject_id);
+CREATE INDEX agmphenotypeannotation_assertedallele_index ON agmphenotypeannotation USING btree (assertedallele_id);
+CREATE INDEX agmphenotypeannotation_inferredallele_index ON agmphenotypeannotation USING btree (inferredallele_id);
+CREATE INDEX agmphenotypeannotation_inferredgene_index ON agmphenotypeannotation USING btree (inferredgene_id);
+
+CREATE TABLE agmphenotypeannotation_gene (
+	agmphenotypeannotation_id bigint,
+	assertedgenes_id bigint
+);
+
+ALTER TABLE agmphenotypeannotation_gene ADD CONSTRAINT agmphenotypeannotation_gene_agmphenotypeannotation_id_fk FOREIGN KEY (agmphenotypeannotation_id) REFERENCES agmphenotypeannotation (id);
+ALTER TABLE agmphenotypeannotation_gene ADD CONSTRAINT agmphenotypeannotation_gene_assertedgenes_id_fk FOREIGN KEY (assertedgenes_id) REFERENCES gene (id);
+CREATE INDEX agmphenotypeannotation_gene_agmphenotypeannotation_index ON agmphenotypeannotation_gene USING btree (agmphenotypeannotation_id);
+
```

Comment:
I think there is a missing index on assertedgenes_id?

Reply:
Good spot - now added